buildSrc/src/main/kotlin/BuildConfig.kt

```diff
@@ -0,0 +1,9 @@
+object BuildConfig {
```

Comment:
I’m a bit on the fence about this approach:
	• having a centralized location for these values does improve visibility and discoverability.
	• on the other hand, since these are Android-specific config versions, I’m not sure there’s significant benefit to separating them out vs. keeping them inline. It adds indirection, and now we’ll need to blame or diff in a separate place, which could impact traceability.

Also, the name might be confusing? It’s very close to `BuildConfig.java`, which is automatically generated by the Android build system. That could lead to misunderstandings.